src/map/water_supply.c

```diff
@@ -75,6 +75,8 @@ void map_water_supply_update_houses(void)
 static void set_all_aqueducts_to_no_water(void)
 {
     int image_without_water = image_group(GROUP_BUILDING_AQUEDUCT_NO_WATER);
+    int highway_image_with_water = assets_get_image_id("Logistics", "Highway_Aqueduct_Full_Left");
+    int highway_image_without_water = assets_get_image_id("Logistics", "Highway_Aqueduct_Empty_Left");
```

Comment:
For a very small performance boost, you can use `static` variables and only get the image id the first time the function is called, even the static variable is still `0`.